<?xml version="1.0"?>
<launch>
<node pkg="cpc_motion_planning" type="nf1_mid_planning_node" name="nf1_mid_planning_node" output="log">
<remap from="/slam_odom" to ="/UgvOdomTopic"/>
<remap from="/set_global_goal" to ="/move_base_simple/goal"/>
</node>
</launch>
